### ISSUE TITLE
Accessiblity Issue Colour Contrast Class .red.white-text

### ISSUE THREAD

**Issue by CalvinRodo** [CSPS-EFPC-DAAN/weasel#1]
The .red.white-text class that can be seen on the following page: 
https://dis-sin.github.io/Evalhalla/?sur=ut1_june18_event

Fails WCAG 1.4.3:
https://dequeuniversity.com/rules/axe/3.2/color-contrast

Here is the link to the WAV Color Contrast Checker: 

https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF&bcolor=F44336

The contrast is okay on large text but it's contrast ratio of 3.6.8:1 is not sufficient for WCAG AA as the contrast has to be at least 4.5:1 and since the user can change the rendering size of the font you need to align to the Normal Text Rules.

You can change the inner text to black to be compliant although it still fails AAA but you only need to be AA compliant so not a big deal.

**Comment by CalvinRodo** [comment]
https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF&bcolor=F44336

**Comment by CalvinRodo** [comment]
Opened in the wrong project.....
